learning_objectives.md

```diff
@@ -32,13 +29,23 @@
 - familiarity with the generation time as a particular type of delay distributions
 - ability to estimate static and time-varying reproduction numbers from time-series data in R
 
+## Nowcasting
+
+- understanding of nowcasting as a particular right truncation problem
+- Ability to perform a simple nowcast in R
+- awareness of the breadth of methods to perform nowcasting
+- R estimation as a nowcasting problem
+
 ## Forecasting
 
 - understanding of forecasting as an epidemiological problem, and its relationship with nowcasting and R estimation
 - understanding of the difference between forecasts, projections and scenarios
 - familiarity with common forecasting models and their properties, and applicability in epidemiology
 - ability to use a common forecasting model on an epidemiological time series in R
-- ability to use a mechanistic model for forecasting an epidemiological time series in R
+- ability to use a semi-mechanistic model for forecasting an epidemiological time series in R
```

Comment:
I changed this as I assumed we would actually extend the renewal model to be used for forecasting.

Reply:
it might be nice to look at the whole breadth, both mechanistic + semi-mechanistic, if we have the time. Where mechanistic ones generally don't have a cone of uncertainty so could discuss impact of strong model prior etc.